### ISSUE TITLE
[FEATURE] Better documentation for providers

### ISSUE THREAD

**Issue by peter-jerry-ye** [wasmCloud/wasmCloud#2957]
### Affected project(s)

- [X] documentation
- [ ] examples
- [ ] wasmCloud host
- [ ] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Is your feature request related to a problem? Please describe.

I'm frustrated when I need to find the configuration of a specific capability provider. If I jump into the [messaging-nats' page](https://github.com/wasmCloud/wasmCloud/pkgs/container/messaging-nats), it's actually the README of wasmcloud, rather than the README of the messaging-nats.

(By the way, I think the configuration of messaging-nats is missing a 's': 'subscriptions' instead of 'subscription')

### Describe the solution you'd like

I would like to see the README of each provider displayed at the package page so that I don't need to dig in the code source's directory to find it.

**Comment by brooksmtownsend** [comment]
@peter-jerry-ye this would be really nice, it's just unfortunately not possible (as far as I know) 😞 The README of a package is tied to the repository that it originates from, rather than a specified README when pushing the package.

If you know of a way to do this with GH packages, please let us know and we can get this taken care of

**Comment by peter-jerry-ye** [comment]
(I assume we are using the container registry) I think there's a 512 character description that can be added according to [the document](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#labelling-container-images). Would it be possible if we can at least put the link to the README there?

**Comment by brooksmtownsend** [comment]
Yep we could do that, we should augment each `wash push` invocation in [provider.yml](https://github.com/wasmCloud/wasmCloud/blob/main/.github/workflows/provider.yml#L96) to also include the annotation above. So for example,
```
wash push <rest of args> -a org.opencontainers.image.description="https://link/to/README"
```
